src/components/AddToCart/AddToCart.js

```diff
@@ -44,27 +44,22 @@ export default function AddToCart({currentMenu}) {
 const AddToCartContainer = styled.div`
   width: 100%;
   height: 3rem;
-  position: sticky;
-  bottom: 0;
-  left: 0;
   font-weight: 700;
-  display: flex;
-  justify-content: space-evenly;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```

Comment:
A very picky note and as a small repetition. You could also write it like this 👇
```suggestion
  grid-template-columns: repeat(2, 1fr);
```